test/extended/router/router.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/openshift/origin/test/extended/util/url"
 )
 
-var _ = g.Describe("[sig-network][Feature:Router][apigroup:operator.openshift.io][apigroup:apps.openshift.io]", func() {
```

Comment:
Is `WaitForRouterServiceIP` the only consumer of both api groups?

Reply:
you are right, I missed this one https://github.com/openshift/origin/blob/master/test/extended/util/framework.go#L1919 . I will have to add the conditional check also to there

Reply:
added